ledger/participant-integration-api/src/main/scala/platform/apiserver/error/ErrorInterceptor.scala

```diff
@@ -52,10 +52,6 @@ final class ErrorInterceptor extends ServerInterceptor {
         * Knowing that Status.INTERNAL is used only by [[com.daml.error.ErrorCategory.SystemInternalAssumptionViolated]],
         * which is marked a security sensitive, we have the following heuristic: check whether gRPC status is Status.INTERNAL
         * and gRPC status description is not security sanitized.
-        * TODO error codes: Consider also checking whether metadata is security sanitized.
-        * TODO error codes: Consider intercepting the exceptions before they are translated in a Status/Metadata pair.
```

Comment:
Created a new story for that: https://digitalasset.atlassian.net/browse/DPP-800 although it might turn we never do it if current implementation proves to be sufficient